packaging/ubuntu-16.04/control

```diff
@@ -14,6 +14,7 @@ Build-Depends: autoconf,
                dh-golang (>=1.7),
                debhelper (>= 9.20160709~) | dh-systemd,
                fakeroot,
+               fuse3 (>= 3.10.5-1) | fuse,
```

Comment:
Does it mean we can drop the dependency on libfuse now?

Reply:
Oops, the dependency is a run time one, not a build time. I've now moved it where it belongs. Bottom line, libfuse is still needed when building.